src/backend/updates.rs

```diff
@@ -83,7 +82,15 @@ fn post_upgrade() {
 fn sync_post_upgrade_fixtures() {}
 
 #[allow(clippy::all)]
-async fn async_post_upgrade_fixtures() {}
+async fn async_post_upgrade_fixtures() {
```

Comment:
It should be in `sync_post_upgrade_fixtures` right?

Reply:
Yep, thanks!